distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Postgres/Postgres_Dialect.enso

```diff
@@ -195,6 +197,24 @@ type Postgres_Dialect
                         Illegal_State.Error "The type computed by our logic is Char, but the Database computed a non-text type ("+db_type.to_display_text+"). This should never happen and should be reported as a bug in the Database library."
             False -> column
 
+    ## PRIVATE
+       Add an extra cast to adjust the output type of certain operations with certain arguments.
+    cast_op_type self (op_kind:Text) (args:(Vector Internal_Column)) (expression:SQL_Expression) =
+        is_bigint ic = ic.sql_type_reference.get.typeid == Types.BIGINT
```

Comment:
personally I prefer to call it `int8` even though `BIGINT` is indeed Postgres's 'primary' name for this. It is just easy to confuse it with Java's `BigInteger` (Postgres's NUMERIC).

Reply:
Done